sources/PanelButton.cpp

```diff
@@ -68,6 +73,14 @@ PanelButton::CreatePanel(int8 which, const BView* target, uint32 flavors,
 		panel = new BFilePanel(B_OPEN_PANEL, &msgr, NULL, flavors, false, NULL,
 			filter);
 
-		panel->Window()->SetTitle(pairs[which].locale);
+		BString title = B_TRANSLATE("Filer");
+		title += ": ";
+
+		if (prefix != NULL)
+			title << prefix << ' ';
```

Comment:
I have just tested it with German, and it seems to work. If it already works for the popup menu, then it should work for the file panels. You don't need to translate it again.